src/main/java/org/example/bookapp/model/Book.java

```diff
@@ -0,0 +1,23 @@
+package org.example.bookapp.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```

Comment:
![image](https://github.com/KhrystynaDidykh/bookApp/assets/164332253/5659e069-0c10-4553-93d5-c3f7c06ee52e)
Put the validation annotations